_posts/2020-12-09-#20477.md

```diff
@@ -10,13 +10,29 @@ status: upcoming
 commit:
 ---
 
-_Notes and questions to follow soon!_
-
-<!-- TODO: Before meeting, add notes and questions
 ## Notes
+* When all inbound connection slots are occupied and a new inbound connection is received, the node attempts to evict an existing inbound connection in order to open up a slot.
+* However, any eviction logic is susceptible to [eclipse attacks](https://www.avivz.net/pubs/15/sec15-paper-heilman_final.pdf). An attacker may try to make lots of new inbound connections to us, and if we always evict another honest node, we will eventually be eclipsed.
```

Comment:
Re eclipse attackts, possibly also link to the review club meetings for [PR 16702](https://bitcoincore.reviews/16702#eclipse-attack) and [PR 17428](https://bitcoincore.reviews/17428).

Reply:
good idea. Done.